config.yaml

```diff
@@ -1,9 +1,21 @@
 options:
+  mount_type:
+    description: |
+      The type of mount to use while mounting an nfs volume. There are two mount
+      types allowed: nfs, efs.
+
+      The efs mount type is specific to AWS EFS mount volumes.
+      The nfs mount type is generic nfs mount which can be used.
+      If no mount type is provided `nfs` is used as the default mount type.
```

Comment:
Why do we have to allow an empty value if the default is `nfs`?

Reply:
The default value is `nfs` so it will not be empty ?

Reply:
You say in the sentence `If no mount type is provided ...` which maps for me to `mount_type = ""`.

Reply:
That is just for the users to let them know that even if they do not provide a value for it `nfs` will be used.